lib/src/cli.rs

```diff
@@ -12,6 +12,7 @@ use ostree_container::store::PrepareResult;
 use ostree_ext::container as ostree_container;
 use ostree_ext::keyfileext::KeyFileExt;
 use ostree_ext::ostree;
+use tracing::field::debug;
```

Comment:
This looks strange, I think you meant to use `debug!` which is a macro, see below

Reply:
Hehheh, yeah that would be from us stumbling through pressing `ctl .` removed :)